Brewfile

```diff
@@ -3,7 +3,7 @@ brew 'dartsim'
 brew 'eigen'
 brew 'libmicrohttpd'
 brew 'ompl'
-brew 'tinyxml2'
+brew 'tinyxml'
 brew 'yaml-cpp'
 
 # Install dartsim dependencies explicitly because, for some unknown reasons,
```

Comment:
Not being able to automatically pull in transitive dependencies is incredibly annoying. It really seems like there should be an obvious way to fix this. Can we add an issue to do so?

@jslee02 do you remember whether this problem was just with Travis CI + Homebrew? Or whether it's a plain Homebrew issue?

In the worst case, it seems like we should be able to do something like `brew deps dartsim` from Travis to generate this list of dependencies that should be installed.

Reply:
Added issue #571 

Maybe we can unblock PRs by pushing this and continue discussion there?

Reply:
Yep, absolutely.

Reply:
> @jslee02 do you remember whether this problem was just with Travis CI + Homebrew? Or whether it's a plain Homebrew issue?

I didn't investigate further but just this workaround. Probably it was just Homebrew. Not sure if it's fixed or now. :disappointed: